data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
@WibbenZ server working stuff should stay in binaries and left lua as a easily customization/server setup or changing the default behaviour, never server working stuff, datapack are supossed to be an example for people build their own datapack, which means server should not have dependency on datapack scripts, otherwise datapack is not an example for a datapack but a server working stuff, and it force people to use default tfs datapack.